README.md

```diff
@@ -60,7 +61,7 @@ You may also want to check out [Meilisearch 101](https://www.meilisearch.com/doc
 
 ## ☁️ Meilisearch cloud
 
-Let us manage your infrastructure so you can focus on integrating a great search experience. Try [Meilisearch Cloud](https://meilisearch.com/pricing) today.
+Let us manage your infrastructure so you can focus on integrating a great search experience. Try [Meilisearch Cloud](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=engine&utm_medium=meilisearch) today.
```

Comment:
Why do we choose a different message from the one added to all integration README.md files?

Reply:
I did not change that message it was written by @davelarkan

Reply:
Indeed the initial message was added when the Cloud first opened for public release. We can change it now if we want something more appropriate?

Reply:
I personally don't have an opinion one way or the other

Reply:
Please go with the one added to the integration so we have something consistent everywhere (it serves the same purposes).

Thanks!

Reply:
done